superdesk/macros/set_default_template_metadata.py

```diff
@@ -72,7 +72,7 @@ def set_default_template_metadata(item, **kwargs):
             to_add.append(sub)
 
     # and now we add the new one
-    item.get('subject').extend([i for i in to_add if i.get('scheme')])
+    item.get('subject', []).extend([i for i in to_add if i.get('scheme') and to_add])
```

Comment:
this wouldn't set anything on item if subject wasn't there before, you can use rather setdefault:
```py
item.setdefault('subject', []).extend(...)
```